README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# SUBSCRIPTIONS CUSTOMER PORTAL
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Local Set-Up
 
-## Available Scripts
+### Create a GitHub account if you do not have one already
 
-In the project directory, you can run:
+- https://github.com/
 
-### `npm start`
+### Fork and Clone the Repo
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Fork the repo into your workspace https://github.com/bold-commerce/subscriptions-customer-portal/tree/shopify
+- Clone your copy of the repo:
+- `$ git clone https://github.com/path/to/your/forked/version`
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Ensure you are on the correct NODE version
 
-### `npm test`
+- `$ nvm install 16`
+- `$ nvm use 16`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Install, build, and run yarn
 
-### `npm run build`
+- `$ yarn`
+- `$ yarn build`
+- `$ yarn start`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Navigate to window
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- `http://localhost:3000`
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### How to setup on a Shopify theme
+- On your shopify theme add this line to the bottom of your `theme.liquid` before the closing `</body></html>` tags
 
-### `npm run eject`
+```html 
+<script src="http://localhost:3000/static/js/bundle.js" type="text/javascript"></script>
+```
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+### Insert div into subscription management page
+- Navigate to the page for managing subscriptions
+- Within the editor menu, click the insert HTML button and insert: 
+    
+    <div id="subscriptions-customer-portal-root"></div>
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Setting up variables
```

Comment:
Add a "Documentation" section at the end of the readme:

## Documentation

For more information, refer to the following resources:

- [Build a Customer Portal](https://developer.boldcommerce.com/default/guides/subscriptions-v2/customer-portals)
- [Bold Subscriptions API Specification](https://developer.boldcommerce.com/default/api/subscriptions)